app/utils/CountryListFactory.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import config.FrontendAppConfig
+import models.Country
+import play.api.Environment
+import play.api.libs.json.Json
+import play.api.libs.json.OFormat.oFormatFromReadsAndOWrites
+import uk.gov.hmrc.govukfrontend.views.viewmodels.select.SelectItem
+
+import javax.inject.{Inject, Singleton}
+
+@Singleton
+class CountryListFactory @Inject() (environment: Environment, appConfig: FrontendAppConfig) {
+
+  private val countryCodesForUkCountries: Set[String] = Set("GB", "UK", "GG", "JE", "IM")
+
+  lazy val countryList: Option[Seq[Country]] = getCountryList
+
+  private def getCountryList: Option[Seq[Country]] = environment.resourceAsStream(appConfig.countryCodeJson) map Json.parse map {
```

Comment:
Can this just be a `Seq[Country]` and we get rid of all the `Option` handling throughout this file and everywhere that it's used?

If the file doesn't exist or can't be read the code won't compile or run but it's our resource sitting in our file system so we shouldn't need worry about that.

Reply:
The return type has now been updated to `Seq[Country]`